users/user-guide-query/multi-stage-query/understanding-multi-stage-explain.md

```diff
@@ -0,0 +1,101 @@
+---
+description: Learn more about multi-stage explain plans and how to interpret them.
+---
+
+# Understanding multi-stage explain plans
+Multi-stage plans are a bit more complex than single-stage plans. 
+This page explains how to interpret multi-stage explain plans.
+
+As explained in [Explaining multi-stage queries](../query-syntax/explain-plan-multi-stage.md), you can use the 
+`EXPLAIN PLAN` syntax to obtain the logical plan of a query.
+There are different formats for the output of the `EXPLAIN PLAN` command, but all of them represent the logical plan 
+of the query.
+
+The query
+```sql
+explain plan for
+select customer.c_address, orders.o_shippriority
+from customer
+join orders
+    on customer.c_custkey = orders.o_custkey
+limit 10
+```
+
+Can produce the following output:
+```
+LogicalSort(offset=[0], fetch=[10])
+  PinotLogicalSortExchange(distribution=[hash], collation=[[]], isSortOnSender=[false], isSortOnReceiver=[false])
+    LogicalSort(fetch=[10])
+      LogicalProject(c_address=[$0], o_shippriority=[$3])
+        LogicalJoin(condition=[=($1, $2)], joinType=[inner])
+          PinotLogicalExchange(distribution=[hash[1]])
+            LogicalProject(c_address=[$4], c_custkey=[$6])
+              LogicalTableScan(table=[[default, customer]])
+          PinotLogicalExchange(distribution=[hash[0]])
+            LogicalProject(o_custkey=[$5], o_shippriority=[$10])
+              LogicalTableScan(table=[[default, orders]])
+```
+
+We can see that each node in the tree represents an operation that is executed in the query and each operator has some
+attributes.
+For example the `LogicalJoin` operator has a `condition` attribute that specifies the join condition and a `joinType`.
+Although some of the attributes shown are easy to understand, some of them may require a bit more explanation.
+
+In our example we can see that the `LogicalTableScan` operator has a table attribute that indicates the table being scanned.
+The table is represented as a list with two elements: the first one is the schema name (`default` by default) and the 
```

Comment:
> schema name (`default` by default)

Should this be "database name" instead or are we using the two terms interchangeably in public documentation?